### PR TITLE
Update Hyperlint Styles

### PR DESCRIPTION
**Pull request by bllchmbrs**
Update Hyperlint styles for #136

### CODE REVIEW

File: .hyperlint/styles/config/vocabularies/hyperlint/accept.txt
Comment (samuelcolvin) [inline .hyperlint/styles/config/vocabularies/hyperlint/accept.txt:1]: I must say I don't love this path, could we not have just one toml file?

**Review by samuelcolvin — COMMENTED** [review]
LGTM, pre-commit is complaining about a missing new line at the end of the file.

File: .hyperlint/styles/config/vocabularies/hyperlint/accept.txt
Comment (bllchmbrs) [reply .hyperlint/styles/config/vocabularies/hyperlint/accept.txt:1]: Thanks for the feedback, it's part of the rules engine right now but I'll talk it over with the team and see what we can do.

**Comment by samuelcolvin** [comment]
replaced by #142.

**Pull request closed without merging**